Scaling.js

```diff
@@ -0,0 +1,13 @@
+import { Dimensions } from "react-native";
+const { width, height } = Dimensions.get("window");
+
+const guidelineBaseWidth = 350;
+const guidelineBaseHeight = 680;
```

Comment:
Could you drop an explanation as to why we are using these exact numbers

Reply:
Guideline sizes are based on standard ~5" screen mobile device